### DISCUSSION TITLE
Swagger generation

### DISCUSSION THREAD

**Discussion by xperiandri** [falcoframework/Falco#74] [Q&A]
Does https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.apiexplorer receive minimal information required for Swagger generation?
How to enrich it?

**Comment by zetashift** [comment]
I also wonder what the best way is to generate Swagger/OpenAPI docs for a JSON API made in Falco!

**Comment by xperiandri** [comment]
@zetashift I've found this tool https://github.com/akhansari/FSharp.OpenApi

**Comment by zetashift** [comment]
Oooh that looks nice.

And when `Fable.Remoting` supports lands, one can generate documentation with that as well

**Comment by jkone27** [comment]
Updates here for openapi for all interested parties: https://github.com/pimbrouwers/Falco/issues/49
